tests/plugins/mentions/manual/mentions.md

```diff
@@ -0,0 +1,10 @@
+@bender-tags: mentions, feature, 4.10.0, 1703
```

Comment:
This manual test deserves to become a generic manual test (so that it gets tested during *each* future release).

To do that you need to remove feature `feature`/`bug` branch.